### PR TITLE
Create directory structure

### PR DESCRIPTION
**Pull request by emapple**
This creates the scaffolding for our basic project structure. In the empty directories I've placed `.readme.md` files that provide a bit more detail.

For now, the `__init__.py` file can remain empty. If we create subdirectories in `src`, each one of those will need its own (empty) `__init__.py` file.

This is the new structure:
```
|--- data            
|     |--- raw            <-- raw data, i.e. text files of books
|
|--- notebooks            <-- notebooks
|
|--- src                  <-- source code
|    |--- __init__.py     <-- makes it a python module
|
|--- setup.py             <-- makes src pip installable
|--- README.md
|--- .gitignore
|--- environment.yml
```

### CODE REVIEW

**Review by diptanil-roy — COMMENTED** [review]
For the raw data, since the file format already includes the last name of the author, I think making a new subdirectory for each author will be redundant. Instead, we can put them all in one folder, and then have a small .py module to access files by the author as and when necessary.

**Comment by emapple** [comment]
@diptanil-roy I'll defer to majority opinion here. I can see arguments either way.

**Review by kdettman — CHANGES_REQUESTED** [review]
The README.md file should include the lines to the following effect:
The `nltk` package requires additional downloads in order to function fully. To do this open python and run the following:
'''
import nltk
nltk.download
'''
A gui window will pop up prompting you to download several files.

**Review by kdettman — APPROVED** [review]

**Comment by diptanil-roy** [comment]
> @diptanil-roy I'll defer to majority opinion here. I can see arguments either way.

That's ok. The only thing is the files are already downloaded, following the scheme we discussed yesterday, and will need another layer of processing, if we have to separate them to different folders. There is no obvious benefit that I can see. But this isn't a major issue.

**Comment by emapple** [comment]
@diptanil-roy then let's put them all together just in `data/raw` and avoid the hassle.

If you're ready, you'll probably want to wait to submit your pull request until this one is approved (and make sure you're up to date with the master branch beforehand).

**Comment by diptanil-roy** [comment]
> @diptanil-roy then let's put them all together just in `data/raw` and avoid the hassle.
> 
> If you're ready, you'll probably want to wait to submit your pull request until this one is approved (and make sure you're up to date with the master branch beforehand).

Yes, I will submit a pull request for the raw data after your directory structure is approved.

File: data/raw/.readme.md
Comment (diptanil-roy) [inline data/raw/.readme.md]: We will just keep all raw .txt files here.

**Review by diptanil-roy — APPROVED** [review]

File: setup.py
Comment (NikhilTilak) [inline setup.py]: Please add all of our names here.

**Review by diptanil-roy — APPROVED** [review]

File: setup.py
Comment (emapple) [reply setup.py]: Added alphabetical by last name, though now am thinking maybe I should have done it in order of DART or KEND

Comment (NikhilTilak) [reply setup.py]: Thats alright. Thanks.

**Pull request merged**
